src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -78,36 +78,35 @@ public ResponseEntity postEvent(@PathVariable final String eventTypeName,
         final HttpServletRequest httpServletRequest = request.getNativeRequest(HttpServletRequest.class);
         final Scope publishingScope = TracingService
                 .activateSpan((Span) httpServletRequest.getAttribute("span"), false);
+        publishingScope.span().setOperationName("publish_events");
+        String sloBucket = "5K-50K";
```

Comment:
make sense to extract to the method and move values to constants
`getSloBucket(httpServletRequest)`